### ISSUE TITLE
简体字／繁体字

### ISSUE THREAD

**Issue by damianodamiani** [readchina/ReadingData#323]
did we decide at any point about using jianti or fanti?

**Comment by duncdrum** [comment]
not to my knowledge, i assume we do whatever the source does?

**Comment by LenaHenningsen** [comment]
Does that create problems if we later on merge data? p.e. if someone has a Lu Xun reference in jianti and someone else in fanti? If that might be the case, I'd be in favor of using jianti thoughtout.

**Comment by damianodamiani** [comment]
I think in our case, as we're not transcribing texts but compiling information, there is no problem with changing fanti into jianti even if the source used fanti (or vice-versa for that matter). So yeah, if @duncdrum says it will help with the final network creation than we could make all in jianti.

**Comment by duncdrum** [comment]
all jianti is fine (please document this in the data dictionary)

**Comment by damianodamiani** [comment]
ok. that also applies to book titles right? everything in jianti?

**Comment by duncdrum** [comment]
yes if we normalise to jianti, we should to do it everywhere in the db
